api/lib/mno_enterprise/concerns/controllers/jpi/v1/admin/account_transactions_controller.rb

```diff
@@ -0,0 +1,33 @@
+module MnoEnterprise::Concerns::Controllers::Jpi::V1::Admin::AccountTransactionsController
+  extend ActiveSupport::Concern
+
+  #==================================================================
+  # Included methods
+  #==================================================================
+  # 'included do' causes the included code to be evaluated in the
+  # context where it is included rather than being executed in the module's context
+  included do
+  end
+
+  #==================================================================
+  # Instance methods
+  #==================================================================
+  # POST /mnoe/jpi/v1/admin/account_transactions
+  def create
+    @account_transaction = MnoEnterprise::AccountTransaction.create!(transaction_params)
```

Comment:
You should add an authorization check on the `@tenant.metadata[:can_manage_organization_credit]`

Reply:
It should also add an AuditEvent